config_examples/config_full.example.json

```diff
@@ -172,7 +172,23 @@
         "jwt_secret_key": "somethingrandom",
         "CORS_origins": [],
         "username": "freqtrader",
-        "password": "SuperSecurePassword"
+        "password": "SuperSecurePassword",
+        "ws_token": "a_secret_ws_token",
+        "enable_message_ws": false
+    },
+    "external_message_consumer": {
+        "enabled": false,
+        "producers": [
+          {
+            "name": "default",
+            "url": "ws://some.freqtrade.bot/api/v1/message/ws",
+            "ws_token": "a_secret_ws_token"
+          }
+        ],
+        "reply_timeout": 10,
```

Comment:
Yes these seem like good defaults for now, maybe further testing can show us otherwise but for a basic system of a few leaders and followers these should work just fine.

Reply:
I think it also really depends on the Strategy you'd be running on the Producer. If it deals with 1h timeframe data, then we'd only get a few messages every hour or so. So a low reply_timeout won't make sense for that.